### ISSUE TITLE
Few pgload.* files are missing from the repo

### ISSUE THREAD

**Issue by zhiboz** [dbuijs/faers2csv#2]
Many thanks for sharing!

`pgload.patient`,  `pgload.rxn`, and `pgload.openfda` mentioned in README.md are not included in the repo. Could you kindly upload those as well?

**Comment by dbuijs** [comment]
Yes. I will add these shortly. I have to remove the database connection details before I add them to the repo.
